CONTRIBUTING.md

```diff
@@ -320,3 +320,14 @@ More information can be found [here](https://docs.aws.amazon.com/cli/latest/user
 cd packages/aws-lambda/layer/bin/
 REGIONS=<region> SKIP_DOCKER_IMAGE=true BUILD_LAYER_WITH=local LAYER_NAME=experimental-instana-nodejs-with-extension ./publish-layer.sh
 ```
+## ESM Support
+
+Use the following command to enable experimental ESM support:
```

Comment:
This is not our public docs, this is for developers. I'd suggest to describe it a little differently such as "We have added ESM support in .... we are still missing native ESM support (ticket ref)....and also put the GH links etc.

Reply:
Updated